ee/tabby-db/migrations/0014_password-reset.up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE password_reset(
+    email VARCHAR(150) PRIMARY KEY NOT NULL,
+    code VARCHAR(36) NOT NULL,
+    created_at TIMESTAMP NOT NULL DEFAULT (DATETIME('now')),
```

Comment:
guess you'll need a status field to represent whether it's used?, e.g `consumed`

Reply:
The entry is removed from the table once it is used